src/System.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/PoolBlockPeriodTest.netcoreapp.cs

```diff
@@ -38,13 +39,13 @@ public void TestAzureBlockingPeriod(string description, object[] Params)
             PoolBlockingPeriodAzureTest(connString, policy);
         }
 
-        [Theory]
-        [InlineData("NonAzure with Default Policy must Enable blocking", new object[] { "nonexistanceserver" })]
-        [InlineData("NonAzure with Auto Policy must Enable Blocking", new object[] { "nonexistanceserver", PoolBlockingPeriod.Auto })]
-        [InlineData("NonAzure with Always Policy must Enable Blocking", new object[] { "nonexistanceserver", PoolBlockingPeriod.AlwaysBlock })]
-        [InlineData("NonAzure with Never Policy must Disable Blocking", new object[] { "nonexistanceserver", PoolBlockingPeriod.NeverBlock })]
-        [InlineData("NonAzure (which contains azure endpoint - nonexistance.WINDOWS.net) with Default Policy must Enable Blocking", new object[] { "nonexistance.windows.net" })]
-        [InlineData("NonAzure (which contains azure endpoint - nonexistance.database.WINDOWS.net.else) with Default Policy must Enable Blocking", new object[] { "nonexistance.database.windows.net.else" })]
+        [ConditionalTheory(nameof(AreConnectionStringsSetup))]
+        [InlineData("NonAzure with Default Policy must Enable blocking", new object[] { "nonexistenteserver" })]
```

Comment:
Nit: Typo, `nonexistenteserver`  should be `nonexistentserver` to be inline with other values.